docs/src/user_guide/system.md

```diff
@@ -0,0 +1,28 @@
+# System
+
+## System description
+
+### Static Components:
+
+We called static components to those that are used for static
+
+- Vector of `Bus` elements, that define all the buses in the network.
+- Vector of `Branch` elements, that define all the branches elements (that connect two buses) in the network.
+- Vector of `StaticInjection` elements, that define all the devices connected to buses that can inject (or withdraw) power.
+- Vector of `PowerLoad` elements, that define all the loads connected to buses that can withdraw current.
+- Vector of `Source` elements, that define source components behind a reactance that can inject or withdraw current.
```

Comment:
I think you could remove `Vector of` from each of these bullets as well as the Dynamic section.  It implies that there is a vector of these things somewhere (there isn't).  

Maybe
```
`Bus` components define all the buses in the network.
```